fakestorage/server.go

```diff
@@ -90,7 +93,7 @@ func NewServerWithOptions(options Options) (*Server, error) {
 		return s, nil
 	}
 
-	s.ts = httptest.NewUnstartedServer(s.mux)
+	s.ts = httptest.NewUnstartedServer(handlers.LoggingHandler(logrus.New().Writer(), s.mux))
```

Comment:
Can you introduce a knob for this and make it off by default? Just want to avoid polluting stderr for projects using fake-gcs-server as a library.

We can add an ``io.Writer`` to ``Options`` for logging  and whenever that's not nil, we setup the LoggingHandler.

Then in the main package, we can pass ``logrus.New().Writer()`` as the writer. That way, users of the library won't get any logs unless they explicitly ask for it, and users of the daemon will get logs. What do you think?

Reply:
👍

Reply:
@fsouza I think a bool flag is enough

Reply:
@cuonglm I can see people asking for the ability to customize where the logs go to (stdout vs stderr, for example) and if we decide to implement that, we'd need yet another parameter, but passing an writer makes that use case possible and avoids the direct dependency on logrus from the library package (along with allowing library users to pass an arbitrary writer if they need an in-memory buffer or something like that for some reason 😁).

Reply:
@fsouza Ah ok, just because I see we're using logrus already. My concern is that we can't pass the io.Writer via command line argument.

Reply:
@fsouza fixed